test/calling_test.js

```diff
@@ -1,459 +1,462 @@
+    });
 
-test('request body is accessible', function() {
-  var params;
-  pretender.post('/some/path/1', function(request) {
-    params = request.requestBody;
+    $.ajax({url: '/some/path?zulu=nation'});
+    equal(params.zulu, 'nation');
   });
 
-  $.ajax({
-    method: 'post',
-    url: '/some/path/1',
-    data: {
-      ok: true
-    }
-  });
-  equal(params, 'ok=true');
-});
+  it('request body is accessible', function() {
+    var params;
+    this.pretender.post('/some/path/1', function(request) {
+      params = request.requestBody;
+    });
 
-test('request headers are accessible', function() {
-  var headers;
-  pretender.post('/some/path/1', function(request) {
-    headers = request.requestHeaders;
+    $.ajax({
+      method: 'post',
+      url: '/some/path/1',
+      data: {
+        ok: true
+      }
+    });
+    equal(params, 'ok=true');
   });
 
-  $.ajax({
-    method: 'post',
-    url: '/some/path/1',
-    headers: {
-      'A-Header': 'value'
-    }
-  });
-  equal(headers['A-Header'], 'value');
-});
+  it('request headers are accessible', function() {
+    var headers;
+    this.pretender.post('/some/path/1', function(request) {
+      headers = request.requestHeaders;
+    });
 
-test('adds requests to the list of handled requests', function() {
-  var params;
-  pretender.get('/some/path', function(request) {
-    params = request.queryParams;
+    $.ajax({
+      method: 'post',
+      url: '/some/path/1',
+      headers: {
+        'A-Header': 'value'
+      }
+    });
+    equal(headers['A-Header'], 'value');
   });
 
-  $.ajax({url: '/some/path'});
-
-  var req = pretender.handledRequests[0];
-  equal(req.url, '/some/path');
-});
+  it('adds requests to the list of handled requests', function() {
+    var params;
+    this.pretender.get('/some/path', function(request) {
+      params = request.queryParams;
+    });
 
-test('increments the handler\'s request count', function() {
-  var handler = function(req) {};
+    $.ajax({url: '/some/path'});
 
-  pretender.get('/some/path', handler);
+    var req = this.pretender.handledRequests[0];
+    equal(req.url, '/some/path');
+  });
 
-  $.ajax({url: '/some/path'});
+  it('increments the handler\'s request count', function() {
+    var handler = function(req) {};
 
-  equal(handler.numberOfCalls, 1);
-});
+    this.pretender.get('/some/path', handler);
 
-test('handledRequest is called', function(assert) {
-  var done = assert.async();
+    $.ajax({url: '/some/path'});
 
-  var json = '{foo: "bar"}';
-  pretender.get('/some/path', function(req) {
-    return [200, {}, json];
+    equal(handler.numberOfCalls, 1);
   });
 
-  pretender.handledRequest = function(verb, path, request) {
-    ok(true, 'handledRequest hook was called');
-    equal(verb, 'GET');
-    equal(path, '/some/path');
-    equal(request.responseText, json);
-    equal(request.status, '200');
-    done();
-  };
+  it('handledRequest is called', function(assert) {
+    var done = assert.async();
```

Comment:
A few lines below you call some assertions from the global scope, like: 

```js
    this.pretender.handledRequest = function(verb, path, request) {
      ok(true, 'handledRequest hook was called');
      equal(verb, 'GET');
      equal(path, '/some/path');
      equal(request.responseText, json);
      equal(request.status, '200');
      done();
    };
```

You may use the `assert` argument in order to keep the test reference, as:

```js
    this.pretender.handledRequest = function(verb, path, request) {
      assert.ok(true, 'handledRequest hook was called');
      assert.equal(verb, 'GET');
      assert.equal(path, '/some/path');
      assert.equal(request.responseText, json);
      assert.equal(request.status, '200');
      done();
    };
```

That makes a exact reference to the current test block and in case you check the reporter or even get a failure, the assertion will reference on the test they were assigned. The global assertions try to determine the current test being executed, this is proved to not go well on some async tests.